usagov/vote-gov-drupal
# Feature/vote 3057/remove po strings

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-3057

## Description

Remove strings from old state page from the po files.

## Deployment and testing

### Post-deploy steps

1. lando retune
2. If necessary - lando import-translations

### QA/Testing instructions

1. View each translated state page and confirm all the components are translating, especially the dates. Note that the state page content will still be in English.
2. Spot check the files against the spreadsheet and confirm only state page strings have been removed.
3. 
## Checklist for the Developer




## Checklist for the Peer Reviewers
